### PR TITLE
Add a styled demo to tooltip v9

### LINKED ISSUE #21997: React-Tooltip: Add demo for styling tooltip
It's not obvious that styling the tooltip requires classNames to be passed to the content slot. Instead of changing the component behavior (as originally), this issue is now about adding a demo to show users how to style a tooltip.

### PR DESCRIPTION
**Pull request by micahgodbolt**
fixes #21997

Add a simple demo showing how to style a tooltip in v9. This came about after customer was having trouble applying styles to the tooltip itself.

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                                                             |         Size (minified/GZIP) |
| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------: |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> | `172.513 kB`<br />`48.34 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr>                            | `32.526 kB`<br />`10.645 kB` |
| <samp>react-tooltip</samp> <br /> <abbr title='bundle-size/Tooltip.fixture.js'>Tooltip</abbr>                                                                                 |  `42.76 kB`<br />`14.701 kB` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/f2ecf6614a11aaf38d2d66bb0f61143487b74e6d'>f2ecf6614a11aaf38d2d66bb0f61143487b74e6d</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)

  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>ButtonNext</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/ButtonNext-mount_base.html">697</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/ButtonNext-mount.html">682</a></td>
            <td>5000</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/ButtonNext-mount.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/TextField-mount_base.html">1668</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/TextField-mount.html">1633</a></td>
            <td>5000</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/TextField-mount.regression.txt">Possible regression</a></td>
           </tr></table><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/Avatar-mount_base.html">1098</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/Avatar-mount.html">1100</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/BaseButton-mount_base.html">1147</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/BaseButton-mount.html">1133</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/Breadcrumb-mount_base.html">3029</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/Breadcrumb-mount.html">3008</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ButtonNext</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/ButtonNext-mount_base.html">697</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/ButtonNext-mount.html">682</a></td>
            <td>5000</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/ButtonNext-mount.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/Checkbox-mount_base.html">1757</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/Checkbox-mount.html">1773</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/CheckboxBase-mount_base.html">1572</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/CheckboxBase-mount.html">1532</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/ChoiceGroup-mount_base.html">5232</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/ChoiceGroup-mount.html">5151</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/ComboBox-mount_base.html">1299</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/ComboBox-mount.html">1240</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/CommandBar-mount_base.html">10955</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/CommandBar-mount.html">10856</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/ContextualMenu-mount_base.html">16089</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/ContextualMenu-mount.html">16541</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/DefaultButton-mount_base.html">1441</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/DefaultButton-mount.html">1377</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/DetailsRow-mount_base.html">4130</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/DetailsRow-mount.html">4165</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/DetailsRowFast-mount_base.html">4081</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/DetailsRowFast-mount.html">4034</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/DetailsRowNoStyles-mount_base.html">3936</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/DetailsRowNoStyles-mount.html">3897</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/Dialog-mount_base.html">2514</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/Dialog-mount.html">2482</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/DocumentCardTitle-mount_base.html">397</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/DocumentCardTitle-mount.html">398</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/Dropdown-mount_base.html">3612</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/Dropdown-mount.html">3592</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderNext</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/FluentProviderNext-mount_base.html">2178</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/FluentProviderNext-mount.html">2172</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/FluentProviderWithTheme-mount_base.html">358</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/FluentProviderWithTheme-mount.html">362</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/FluentProviderWithTheme-virtual-rerender_base.html">331</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/FluentProviderWithTheme-virtual-rerender.html">324</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">407</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/FluentProviderWithTheme-virtual-rerender-with-unmount.html">377</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/FocusTrapZone-mount_base.html">2021</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/FocusTrapZone-mount.html">2074</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/FocusZone-mount_base.html">2053</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/FocusZone-mount.html">2124</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/IconButton-mount_base.html">2053</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/IconButton-mount.html">2049</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/Label-mount_base.html">570</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/Label-mount.html">566</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/Layer-mount_base.html">3275</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/Layer-mount.html">3246</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/Link-mount_base.html">689</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/Link-mount.html">707</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/MakeStyles-mount_base.html">1867</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/MakeStyles-mount.html">1865</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/MenuButton-mount_base.html">1749</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/MenuButton-mount.html">1745</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/MessageBar-mount_base.html">2378</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/MessageBar-mount.html">2396</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/Nav-mount_base.html">3614</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/Nav-mount.html">3644</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/OverflowSet-mount_base.html">1352</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/OverflowSet-mount.html">1349</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/Panel-mount_base.html">2416</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/Panel-mount.html">2371</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/Persona-mount_base.html">1063</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/Persona-mount.html">1099</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/Pivot-mount_base.html">1671</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/Pivot-mount.html">1660</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/PrimaryButton-mount_base.html">1566</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/PrimaryButton-mount.html">1574</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/Rating-mount_base.html">8085</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/Rating-mount.html">8073</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/SearchBox-mount_base.html">1546</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/SearchBox-mount.html">1580</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/Shimmer-mount_base.html">2780</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/Shimmer-mount.html">2760</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/Slider-mount_base.html">2180</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/Slider-mount.html">2218</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/SpinButton-mount_base.html">5340</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/SpinButton-mount.html">5448</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/Spinner-mount_base.html">645</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/Spinner-mount.html">667</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/SplitButton-mount_base.html">3451</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/SplitButton-mount.html">3427</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/Stack-mount_base.html">752</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/Stack-mount.html">763</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/StackWithIntrinsicChildren-mount_base.html">2527</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/StackWithIntrinsicChildren-mount.html">2538</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/StackWithTextChildren-mount_base.html">5589</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/StackWithTextChildren-mount.html">5593</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/SwatchColorPicker-mount_base.html">12142</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/SwatchColorPicker-mount.html">12072</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/TagPicker-mount_base.html">2911</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/TagPicker-mount.html">2863</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/TeachingBubble-mount_base.html">98473</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/TeachingBubble-mount.html">98780</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/Text-mount_base.html">697</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/Text-mount.html">646</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/TextField-mount_base.html">1668</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/TextField-mount.html">1633</a></td>
            <td>5000</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/TextField-mount.regression.txt">Possible regression</a></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/ThemeProvider-mount_base.html">1434</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/ThemeProvider-mount.html">1416</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/ThemeProvider-virtual-rerender_base.html">878</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/ThemeProvider-virtual-rerender.html">913</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">2102</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">2138</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/Toggle-mount_base.html">1017</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/Toggle-mount.html">1072</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/buttonNative-mount_base.html">360</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22000/perf-test/buttonNative-mount.html">365</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

File: packages/react-tooltip/src/stories/TooltipStyled.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+
+import { Tooltip, TooltipProps } from '../Tooltip';
+import { Button } from '@fluentui/react-button';
+import { SlideTextRegular } from '@fluentui/react-icons';
+import { makeStyles } from '@griffel/react';
+
+const getStyles = makeStyles({
+  toolTip: {
```
Comment (layershifter) [inline packages/react-tooltip/src/stories/TooltipStyled.stories.tsx:9]:
```suggestion
  tooltip: {
```

nit: ["tooltip"](https://en.wiktionary.org/wiki/tooltip) is a word AFAIR

File: packages/react-tooltip/src/stories/TooltipStyled.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    backgroundColor: 'green',
+    color: 'white',
```
Comment (layershifter) [inline packages/react-tooltip/src/stories/TooltipStyled.stories.tsx:11]: Can we use `tokens` instead to ensure that customers use good patterns based on examples?

File: packages/react-tooltip/src/stories/TooltipStyled.stories.tsx (hunk 1 above)
Comment (micahgodbolt) [reply packages/react-tooltip/src/stories/TooltipStyled.stories.tsx:9]: hah...yeah, not sure why i wrote it that way

File: packages/react-tooltip/src/stories/TooltipStyled.stories.tsx (hunk 2 above)
Comment (micahgodbolt) [reply packages/react-tooltip/src/stories/TooltipStyled.stories.tsx:11]: i debated on tokens or not, my goal was to keep the demo incredibly generic, and not introduce more imports and concepts. Was trying to think of a style someone might do that didn't use shorthands, tokens, or anything like that.

File: packages/react-tooltip/src/stories/TooltipStyled.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  },
+});
+
+export const Styled = (props: Partial<TooltipProps>) => {
+  const styles = getStyles();
+  return (
+    <Tooltip
+      withArrow
+      content={{ children: 'Example tooltip', className: styles.toolTip }}
```
Comment (micahgodbolt) [inline packages/react-tooltip/src/stories/TooltipStyled.stories.tsx:20]:
```suggestion
      content={{ children: 'Example tooltip', className: styles.tooltip }}
```

File: packages/react-tooltip/src/stories/TooltipStyled.stories.tsx (hunk 2 above)
Comment (micahgodbolt) [reply packages/react-tooltip/src/stories/TooltipStyled.stories.tsx]: Pulled in tokens and it doesn't feel like it jumped in complexity too much. I'm good with it

**Review by khmakoto — APPROVED** [review]

**Pull request merged**